### PR TITLE
dev-python/pycdio: bump version to 2.1.0

### PR DESCRIPTION
**Pull request by CaseOf**
dev-python/pycdio has been updated upstream to 2.1.0 that is now supporting Python 3.

### CODE REVIEW

**Comment by flappyports** [comment]
@CaseOf, wow thank you for the quick submission. Please check my comments for a few minor fixes and then I will merge.

Again, thanks!

**Comment by CaseOf** [comment]
@flappyports which comments? where are they?

**Comment by flappyports** [comment]
@CaseOf, sorry, I failed to submit them and lost the comments on review. Give me a few to look it over again. If not tonight, then in the morning.

**Comment by CaseOf** [comment]
oh, ok.

**Comment by CaseOf** [comment]
@flappyports so?

**Comment by CaseOf** [comment]
@flappyports  ping again.

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		example/*.py || die
+
+	# Disable failing tests.
+	sed -i -e "s/test_get_set/_&/" test/test-cdtext.py || die
+	sed -i -e "s/test_fs/_&/" test/test-isocopy.py || die
+	distutils-r1_python_prepare_all
+}
+
+python_install_all(){
+	use examples && local EXAMPLES=( example/. )
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: this doesn't work

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+DEPEND="
+	${RDEPEND}
+	dev-lang/swig
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+RESTRICT="test"  # currently tests fail
+
+python_prepare_all() {
+	append-cflags -fno-strict-aliasing
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: I bet this is only needed for py2

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: remove py2 while at it pls

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND=">=dev-libs/libcdio-2.0.0"
+
+DEPEND="
+	${RDEPEND}
+	dev-lang/swig
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: this needs to be in `BDEPEND`

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: not needed anymore

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+RESTRICT="test"  # currently tests fail
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: please try and fix these

**Comment by CaseOf** [comment]
Thanks for your review 🙂

**Comment by CaseOf** [comment]
just pushed requested changes

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 flag-o-matic
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: `flag-o-matic` not used anymore

File: dev-python/pycdio/pycdio-2.1.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+	sed -i -e "s/test_fs/_&/" test/test-isocopy.py || die
+	distutils-r1_python_prepare_all
+}
+
+python_install_all(){
+	distutils-r1_python_install_all
+	if use examples; then
+		docinto examples
+		dodoc example/*
+	fi
```
Comment (SoapGentoo) [inline dev-python/pycdio/pycdio-2.1.0.ebuild]: make this block:
```
	if use examples; then
		docinto examples
		dodoc -r example/.
		docompress -x /usr/share/doc/${PF}/examples
	fi
```
Rationale: Avoid globs if possible

**Comment by CaseOf** [comment]
pushed too

**Pull request closed without merging**